src/Search/SearchWithSuggestions/Suggestions.js

```diff
@@ -76,9 +76,11 @@ const Suggestions = ({
       ))}
     </>
   ) : (
-    <div className={styles.suggestion + ' ' + styles.noresults}>
-      {!isSearching ? 'No results found' : 'Searching...'}
-    </div>
+    !searchTerm.includes(',') && (
```

Comment:
This check breaks all searches that might include the `,` symbol, therefore `multipleCheck` condition should be added as well